server/src/main/java/org/elasticsearch/cluster/metadata/AliasMetaData.java

```diff
@@ -41,7 +41,7 @@
 
 import static java.util.Collections.emptySet;
 
-public class AliasMetaData extends AbstractDiffable<AliasMetaData> {
+public class AliasMetaData extends AbstractDiffable<AliasMetaData> implements ToXContent{
```

Comment:
can we go for either ToXContentFragment or ToXContentObject?

Reply:
:sweat_smile: